README.md

```diff
@@ -57,18 +57,20 @@ Monero. See [here](https://github.com/fireice-uk/cryptonote-speedup-demo/) for d
 
 If your prefered coin is not listed, you can choose one of the following algorithms:
 - 256Kib scratchpad memory
-    - cryptonight_turtle
+    - cryptonight_turtle (equal to cryptonight_v8 but `0x1FFF0` as mask, 1/8 of iterations and scratchpad size)
 - 1MiB scratchpad memory
     - cryptonight_lite
     - cryptonight_lite_v7
     - cryptonight_lite_v7_xor (algorithm used by ipbc)
 - 2MiB scratchpad memory
     - cryptonight
+    - cryptonight_gpu (for Ryo 02/2019 fork)
     - cryptonight_masari (used in 2018)
     - cryptonight_v7
     - cryptonight_v7_stellite
     - cryptonight_v8
-    - cryptonight_v8_half (used by masari and stellite)
+    - cryptonight_v8_half (used by masari and stellite; equal to cryptonight_v8 but with half iterations)
+    - cryptonight_v8_zelerius (equal to cryptonight_v8 but with 393216 iteration)
```

Comment:
Same here. Algo is not a coin. If they can't be bothered to come up with original algo let's just call it `cryptonight_v8_half`. I don't think annotations are needed - just internal mappings will be enough

Reply:
`cryptonight_v8_half` is an other algorithm with the half iteration count of v8. This algo uses an different iteration count.